### PR TITLE
[AArch64] Fix sched model of Neoverse N2

### PR DESCRIPTION
**Pull request by FLZ101**
* fix write order of "Load vector reg, immed post-index"
* fix a typo

### CODE REVIEW

**Review by davemgreen — APPROVED** [review]
LGTM. I must have missed this one as it is WriteI not WriteAdr

**Comment by FLZ101** [comment]
@c-rhodes @sjoerdmeijer @rj-jesus

File: llvm/lib/Target/AArch64/AArch64SchedNeoverseN2.td [hunk 1]
```diff
@@ -858,7 +858,7 @@ def : InstRW<[N2Write_6cyc_1L], (instregex "^LDR[SDQ]l$",
                                            "^LDUR[BHSDQ]i$")>;
 
 // Load vector reg, immed post-index
-def : InstRW<[N2Write_6cyc_1I_1L, WriteI], (instregex "^LDR[BHSDQ]post$")>;
```
Comment (rj-jesus) [inline llvm/lib/Target/AArch64/AArch64SchedNeoverseN2.td:861]: Thanks, LGTM. Maybe we could merge this with the pre-index form below and use WriteAdr, what do you think?

Comment (FLZ101) [reply llvm/lib/Target/AArch64/AArch64SchedNeoverseN2.td:861]: Sure.

Comment (FLZ101) [reply llvm/lib/Target/AArch64/AArch64SchedNeoverseN2.td:861]: Done.

Comment (rj-jesus) [reply llvm/lib/Target/AArch64/AArch64SchedNeoverseN2.td:861]: Thanks, LGTM!

**Comment by FLZ101** [comment]
@davemgreen Can this PR be merged？

**Comment by davemgreen** [comment]
Can you rebase? I updated some cyc_'s to c_' to try and keep them consistent between models. Sorry about the conflict.

**Comment by FLZ101** [comment]
> Can you rebase? I updated some cyc_'s to c_' to try and keep them consistent between models. Sorry about the conflict.

The rebase is done.

**Comment by davemgreen** [comment]
Thanks.

**Pull request merged**